src/org/labkey/targetedms/query/ModificationManager.java

```diff
@@ -156,6 +157,40 @@ public void exec(ResultSet rs) throws SQLException
         return new SqlSelector(TargetedMSManager.getSchema(), sql).getArrayList(PeptideSettings.RunStructuralModification.class);
     }
 
+    public static List<PeptideSettings.IsotopeModification> getIsotopeModificationsUsedInRun(int runId)
```

Comment:
Can you add comments to the "UsedInRun" and "ForRun" variants of the methods to help distinguish them? It's mostly apparent from the method names, but since it's such a subtle difference would be good to be a bit more explicit.

Reply:
Comments added in commit https://github.com/LabKey/targetedms/commit/3ea4aa9a058c4bf38cbd107c3af7cf85bc30e16e